client/src/templates/Challenges/redux/execute-challenge-saga.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```

Comment:
This line is throwing errors for me locally.

Reply:
I wondered what was going on. This was an accident on my part, because GitPod had a meltdown when I was trying to install Cypress.